compose/django/Pipfile

```diff
@@ -47,6 +47,9 @@ Django = "==2.0.9"
 PyJWT = "==1.6.4"
 pyOpenSSL = "==17.5.0"
 uWSGI = "==2.0.17.1"
+fpdf = "==1.7.2"
```

Comment:
`fpdf`, `pillow`, and `minio` all look to be utilized in development/testing only.

These should be moved to `[dev-packages]`.